Formula/k/kamel.rb

```diff
@@ -44,7 +44,9 @@ def install
     assert_match version.to_s, version_output
 
     run_output = shell_output("echo $(#{bin}/kamel run 2>&1)")
-    assert_match "Error: run expects at least 1 argument, received 0", run_output
+    assert_match "Error: " \
+                 "invalid configuration: " \
+                 "no configuration has been provided, try setting KUBERNETES_MASTER environment variable", run_output
```

Comment:
I just removed this test in https://github.com/Homebrew/homebrew-core/pull/159288

Reply:
Should the test just stay removed or should i keep the current PR opened to follow up the #159288 after it is merged ?

Reply:
this pr would be closed by https://github.com/Homebrew/homebrew-core/pull/159288 per merge